components/src/form-field/error-summary.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from "@angular/core";
```

Comment:
This count isn't working in the Storybook, is that just because we're mocking the i18nService?
![Screen Shot 2022-05-27 at 2 43 43 pm](https://user-images.githubusercontent.com/31796059/170630745-120c9ae2-cb5e-4219-a351-00e03bab955f.png)

Reply:
Also, we need singular wording if only 1 field requires attention (so we don't have "1 fields")

Reply:
Yes, the mock doesn't handle substitutions. (Which I think is fine)